### PR TITLE
oonf_api: remove trailing white spaces

### PR DESCRIPTION
**Pull request by OlegHahm**
introduced in ac0e2f4371b471b6f9c0b2020559469b61cd78c4

### CODE REVIEW

**Comment by authmillenon** [comment]
ACK.

**Comment by Kijewski** [comment]
NACK. The whitespaces are part of the unified diff format.

**Comment by authmillenon** [comment]
@Kijewski git can ignore whitespaces in patch files.

**Comment by authmillenon** [comment]
I tested it and it works so I still say ACK.

**Comment by Kijewski** [comment]
Diff does not choke on this invalid input, but it is nevertheless wrong.
Context lines are denoted by a space, same as inserted lines got a plus and removed lines a minus.

**Comment by authmillenon** [comment]
I revoke my ACK because there are other changes than removed whitespaces here.

**Comment by authmillenon** [comment]
@OlegHahm please explain.

**Comment by OlegHahm** [comment]
@authmillenon, which changes are you talking about?

**Comment by OlegHahm** [comment]
> Context lines are denoted by a space, same as inserted lines got a plus and removed lines a minus.

Do you have a reference?

**Comment by Kijewski** [comment]
"The lines common to both files begin with a space character." (http://www.gnu.org/software/diffutils/manual/html_node/Detailed-Unified.html#Detailed-Unified) Empty lines are not mentioned.

**Comment by OlegHahm** [comment]
The same manual offers also an option for omitting these trailing spaces:
http://www.gnu.org/software/diffutils/manual/html_node/Trailing-Blanks.html

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 1]
```diff
@@ -83,7 +83,7 @@ index 77c519b..37e77ac 100644
 @@ -51,6 +51,12 @@
  #include <winsock2.h>
  #endif
- 
+
 +#ifdef RIOT_VERSION
 +int getpagesize(void) {
 +	return 512;
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 2]
```diff
@@ -91,7 +91,7 @@ index 77c519b..37e77ac 100644
 +#endif
 +
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 3]
```diff
@@ -116,7 +116,7 @@ index 103c88f..8a6cd2b 100644
 @@ -48,7 +48,7 @@
  #include "common/common_types.h"
  #include "common/daemonize.h"
- 
+
 -#ifndef WIN32
 +#if (!defined(_WIN32)) && (!defined(RIOT_VERSION))
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 4]
```diff
@@ -138,7 +138,7 @@ index dedab2c..1b602ec 100644
 +#include "inet_pton.h"
 +#define DONT_HAVE_SIN6_SCOPE_ID
 +#endif
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 5]
```diff
@@ -165,14 +165,14 @@ index dedab2c..1b602ec 100644
 +  dst->_type = (uint8_t)src->std.sin6_family;
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

File: pkg/oonf_api/0001-add-RIOT-support.patch [hunk 6]
```diff
@@ -165,14 +165,14 @@ index dedab2c..1b602ec 100644
...
    return 0;
  }
- 
+
 @@ -204,12 +211,12 @@ netaddr_from_socket(struct netaddr *dst, const union netaddr_socket *src) {
  int
  netaddr_to_socket(union netaddr_socket *dst, const struct netaddr *src) {
    /* copy address type */
 -  dst->std.sa_family = src->_type;
 +  dst->std.sin6_family = src->_type;
```
Comment (authmillenon) [inline pkg/oonf_api/0001-add-RIOT-support.patch]: This one

**Comment by authmillenon** [comment]
And many more.

**Comment by OlegHahm** [comment]
@authmillenon, please look again. You're not commenting on my changes.

**Comment by authmillenon** [comment]
OMFG, reviewing patches is a nightmare. NVM

**Comment by authmillenon** [comment]
@Kijewski are you revoking your NACK? We stripped the whitespaces in the past with no problems, so I don't understand why this is suddenly now relevant. (Also: AFAIK `git-diff` (used for these patches) is independent from `diffutils`, so your quote would be not valid)

**Comment by authmillenon** [comment]
Basically superseded by #3882 (it removes the whitespaces, too)

**Comment by authmillenon** [comment]
(and changes all the patches)

**Pull request closed without merging**